.github/workflows/tests.yml

```diff
@@ -26,6 +26,9 @@ jobs:
       - name: Get the archive from the url.
         run: curl -OL ${{ env.ROC_RELEASE_URL }}
 
+      - name: store short commit SHA
+        run: echo "SHORT_COMMIT_SHA=$(ls | grep roc_nightly | cut -d '-' -f 6 | cut -d '.' -f 1)"  >> $GITHUB_ENV
```

Comment:
To be able to get necessary dependencies like static-site-gen with the same commit as the nightly.